mono/utils/mono-state.h

```diff
@@ -0,0 +1,30 @@
+/**
+ * \file
+ * Support for interop with the Microsoft Error Reporting tool (header)
```

Comment:
the state protocol isn't strictly tied to MERP right? I'd not mention it here then. The description you have in the .c is better.

Reply:
It's not any more, you're right. I'll clean that up.